admin/partials/boldgrid-backup-admin-support.php

```diff
@@ -77,12 +77,12 @@
 		wp_kses(
 			/* translators: 1: HTML anchor for the URL address for BoldGrid Central, 2: HTML anchor close tag.. */
 			__(
-				'You can receive premium support using %1$sBoldGrid Central%2$s',
+				'Create a ticket at %1$ssupport.boldgrid.com%2$s',
```

Comment:
Please change this to something like this:
`Create a ticket with %1$sBoldGrid Support%2$s`
rather than having 'support.boldgrid.com' be the link text.